### PR TITLE
Fix gizmo position after scene edit

### PR DESCRIPTION
**Pull request by slimbuck**
This PR:
- recalculates the gizmo position upon scene edit to keep it centered on the splat
- update npm dependencies, including latest engine
- bump app to v0.15.0

### CODE REVIEW

File: src/splat.ts [hunk 1]
```diff
@@ -84,6 +78,31 @@ class Splat extends Element {
...
+
+        // update splat data local bound
+        const opacity = this.splatData.getProp('opacity');
+        this.splatData.calcAabb(localBound, (i: number) => {
+            return opacity[i] > -1000;
+        });
+
+        // calculate meshinstance aabb (transformed local bound)
+        const meshInstance = this.root.gsplat.instance.meshInstance;
+        meshInstance._aabb.setFromTransformedAabb(localBound, this.entity.getWorldTransform());
```
Comment (willeastcott) [inline src/splat.ts:94]: No way to avoid non-API calls?

**Review by willeastcott — APPROVED** [review]

File: src/splat.ts (hunk 1 above)
Comment (slimbuck) [reply src/splat.ts:94]: I wasn't able to use public API. This needs addressing with @mvaligursky.

Comment (willeastcott) [reply src/splat.ts:94]: Is there an engine issue. Just wanna make sure we don't forget this.

Comment (slimbuck) [reply src/splat.ts:94]: Good point, I've created https://github.com/playcanvas/engine/issues/6055.

**Pull request merged**